app/controllers/automated_tests_controller.rb

```diff
@@ -36,15 +36,15 @@ def update
     # Perform transaction, if errors, none of new config saved
     @assignment.transaction do
       @assignment = process_test_form(@assignment, assignment_params)
-      # begin
-      #   # Process testing framework form for validation
-      #   @assignment = process_test_form(@assignment, params)
-      # rescue Exception, RuntimeError => e
-      #   @assignment.errors.add(:base, I18n.t("assignment.error",
-      #                                        :message => e.message))
-      #   render :manage
-      #   return
-      # end
+      begin
```

Comment:
Delete this block. There is already a call to process_test_form above. If there's an error, let it blow up for now.